dials_data/definitions/ssx_CuNiR_test_data.yml

```
@@ -0,0 +1,14 @@
+name: SSX CuNiR test data
```

Recommend dropping `test` and `data` from the file name, you could use a `_processed` suffix as in https://github.com/dials/data/blob/master/dials_data/definitions/mpro_x0305_processed.yml

Name could be
```suggestion
name: DIALS output of processing a SSX CuNiR data set
```
to distinguish from raw data